articles/quickstart/backend/java-spring-security5/download.md

```diff
@@ -0,0 +1,23 @@
+The sample can be run from the command line as follows:
+
+Linux / macOS:
+```bash
+./gradlew clean bootRun
+```
+
+Windows:
+```bash
+gradlew.cmd clean bootRun
+```
+
+The sample includes a [Docker](https://www.docker.com) image ready to run with the following command:
+
+```bash
+# Linux / macOs
+sh exec.sh
+```
+
+```bash
+# Windows
+./exec.ps1
+```
```

Comment:
I'd add where I could find it exposed. e.g. `localhost:8080`. Note that docker might expose it on a different port.

Reply:
New commit to add this info to the Download instructions.